apps/assistants/views.py

```diff
@@ -90,11 +98,38 @@ class CreateOpenAiAssistant(BaseOpenAiAssistantView, CreateView):
     button_text = "Create"
     permission_required = "assistants.add_openaiassistant"
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        if "file_formset" not in context:
+            context["file_formset"] = self._get_file_formset()
+        return context
+
+    def _get_file_formset(self):
```

Comment:
nit: since this is calling 1 function, you might as well call `get_file_formset` directly

Reply:
I'm going to leave this since this is called in 2 places and it does simplify a bit by not requiring you to pass the request